common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAuthorizationRequest.java

```diff
@@ -128,6 +132,7 @@ protected MicrosoftAuthorizationRequest(final Builder builder) {
         mCodeChallengeMethod = mPkceChallenge.getCodeChallengeMethod();
         mCodeChallenge = mPkceChallenge.getCodeChallenge();
         mState = generateEncodedState();
+        mClientInfoEnabled = "1";
```

Comment:
can you add some comment specifying what '1' stands for (and what are the possible values)?